src/super_gradients/training/transforms/transforms.py

```diff
@@ -826,13 +828,52 @@ def __init__(self, prob: float, max_targets: Optional[int] = None):
 
     def __call__(self, sample):
         image, targets = sample["image"], sample["target"]
+        crowd_targets = sample.get("crowd_targets")
         if len(targets) == 0:
             targets = np.zeros((0, 5), dtype=np.float32)
-        boxes = targets[:, :4]
-        image, boxes = _mirror(image, boxes, self.prob)
-        targets[:, :4] = boxes
+        if random.random() < self.prob:
+            image = _flip_horizontal_image(image)
+            boxes = targets[:, :4]
+            _, width, _ = image.shape
+            boxes = _flip_horizontal_boxes(boxes, width)
+            targets[:, :4] = boxes
```

Comment:
You can actually simplify
this
```
boxes = targets[:, :4]
_, width, _ = image.shape
boxes = _flip_horizontal_boxes(boxes, width)
targets[:, :4] = boxes
```

Into this:
```
_, width, _ = image.shape
targets[:, :4] = _flip_horizontal_boxes(targets[:, :4], width)
```

Which in my opinion quite easier to understand & more compact